linux_os/guide/services/nfs_and_rpc/nfs_configuring_servers/use_kerberos_security_all_exports/bash/shared.sh

```diff
@@ -0,0 +1,19 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+nfs_exports=()
+readarray -t nfs_exports < <(grep -E "^/.*[[:space:]]+ .*\(.*\)[[:space:]]*$" /etc/exports | awk '{print $2}')
+
+for nfs_export in "${nfs_exports[@]}"
+do
+    correct_export=""
+    if [ "$(grep -c "sec=" <<<"$nfs_export")" -eq 0 ]; then
```

Comment:
You probably swap the command in the `if` branch with the command in the `else` branch. If the count of `sec=` in the string is 0, it won't make sense to try replacing `sec=` by sed.

Reply:
should be fixed in a59ed41